### DISCUSSION TITLE
How to configure Nvchad for snap installed version  of neovim?

### DISCUSSION THREAD

**Discussion by leamandeep** [NvChad/NvChad#2394] [Q&A] [answered]
My question is how to configure to nvchad for snap-installed package. I have gone through installation section in docs and didn't find any solution for snap installed version of neovim. Snap installed version of neovim is missing config file. I don't know what to do? Please guide me on this problem.

Thanks in advance

![Screenshot_from_2023-09-20_18-27-06.png](https://github.com/NvChad/NvChad/assets/68345524/4e26378d-1f4c-4e06-9a77-617c17cc95c8)

**Comment by siduck** [comment, marked as answer]
@elamandeep  install nvchad , check the install docs. and I strongly advise against snaps so either use this https://github.com/MordechaiHadad/bob or some ppa
